rflx/pyrflx/pyrflx.py

```diff
@@ -33,6 +26,21 @@ def __init__(
                     m, model.refinements, skip_message_verification
                 )
 
+    @classmethod
+    def parse(
+        cls,
+        files: List[str],
```

Comment:
For parameters preferably `Sequence` should be used, as this is less restrictive.

Reply:
Fixed by https://github.com/Componolit/RecordFlux/pull/468/commits/f4d5340c5ac47e139227bf5e76a50f4f7f681d29.